src/NzbDrone.Core.Test/NotificationTests/GotifyTests/GotifyServiceFixture.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Linq;
+using Moq;
+using NUnit.Framework;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Movies;
+using NzbDrone.Core.Notifications;
+using NzbDrone.Core.Notifications.Gotify;
+using NzbDrone.Core.Test.Framework;
+
+namespace NzbDrone.Core.Test.NotificationTests
+{
+    [TestFixture]
+    public class GotifyServiceFixture : CoreTest<Gotify>
+    {
+        [SetUp]
+        public void Setup()
+        {
+            Subject.Definition = new NotificationDefinition
+            {
+                Settings = new GotifySettings
+                {
+                    Server = "https://example.invalid",
+                    AppToken = "token",
+                    Priority = 5,
+                    IncludeMoviePoster = false,
+                    IncludeInstanceNameInTitle = false,
+                    MetadataLinks = Enumerable.Empty<int>(),
+                    PreferredMetadataLink = (int)MetadataLinkType.Tmdb
+                }
+            };
+        }
+
+        [TestCase(false, "MyRadarr", false)]
+        [TestCase(true,  "MyRadarr", true)]
+        [TestCase(true,  "",         false)]
+        [TestCase(true,  "   ",      false)]
```

Comment:
Don't format arguments.

```suggestion
        [TestCase(false, "MyRadarr", false)]
        [TestCase(true, "MyRadarr", true)]
        [TestCase(true, "", false)]
        [TestCase(true,  "   ", false)]
```